content/agent/prometheus.md

```diff
@@ -54,9 +54,9 @@ annotations:
 
 ### From custom to official integration
 
-By default all metrics retrieved by the generic prometheus check are considered custom metrics, if you are monitoring an off-the-shelf software and you think it would deserve an official integration, don't hesitate to contribute.
+By default all metrics retrieved by the generic Prometheus check are considered custom metrics. If you are monitoring off-the-shelf software and think it deserves an official integration, don't hesitate to contribute.
```

Comment:
… official integration, _please_ don't hesitate to contribute.

Minor nit, but when making a request (even hypothetically), it always pays to be polite. 😄